setup.py

```diff
@@ -27,8 +27,10 @@
 with open("README.rst") as f:
     LONG_DESCRIPTION = "".join(f.readlines())
 
+print(find_packages(exclude=["tests*", "examples*", "tutorials*"]))
+
```

Comment:
Do we want to add this print?

Reply:
Not at all, just a little leftover debugging from the 400+ changed files